lib/Drush/CommandFiles/core/DrupliconCommands.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Drush\CommandFiles\core;
+
+class DrupliconCommands {
+
+  /**
+   * Print druplicon as post-command output.
+   *
+   * @hook post-command *
+   * @option $druplicon Shows the druplicon as glorious ASCII art.
+   */
+  public function druplicon($result, $argsAndOptions, $annotationData) {
+    if (isset($argsAndOptions['options']['druplicon'])) {
```

Comment:
Sure, split them out. Or make some sort of $request object with $getArguments() and getOptions() methods.